internal/services/dashboard/dashboard_grafana_resource_test.go

```diff
@@ -149,7 +154,8 @@ resource "azurerm_dashboard_grafana" "test" {
   public_network_access_enabled     = false
 
   identity {
-    type = "SystemAssigned"
+    type = "UserAssigned"
```

Comment:
Can you align the `=` here, we'll be re-enabling the linter for this soon so should get ahead of fixes.

Reply:
Sure. Fixed.